doc/source/whatsnew/v0.24.2.rst

```diff
@@ -51,7 +51,7 @@ Bug Fixes
 
 **I/O**
 
--
+- Better handling of terminal printing when the terminal dimensions are not known (#25080);
```

Comment:
use `` (:issue:`25080`) ``

Reply:
Man, I'm extremely sorry, I only saw this now! Making changes

Reply:
There. Are there any other changes I should make?